src/products/views/ProductVariant.tsx

```diff
@@ -146,6 +144,7 @@ export const ProductVariant: React.FC<ProductUpdateProps> = ({
                     }}
                     onVariantClick={variantId => {
                       navigate(productVariantEditUrl(productId, variantId));
+                      setErrors([]);
```

Comment:
I'd rather set it in `React.useEffect` - now errors persist if you go back using browser history.